go.mod

```diff
@@ -4,8 +4,9 @@ go 1.16
 
 replace (
 	github.com/ethereum/go-ethereum => github.com/prysmaticlabs/bazel-go-ethereum v0.0.0-20210222122116-71d15f72c132
+	github.com/ferranbt/fastssz => github.com/atif-konasl/fastssz v0.0.0-20210705113036-087ec0cbb160
```

Comment:
What is the motivation of this change?
IMHO we should not change it to your repository. Push it to lukso namespace or rethink if we need this replacement at all

Reply:
Agree, lukso-network namespace required